Pull request

Wolfvak opened the pull request.

Wolfvak committed:
Everything loads from "/injector/" now

Added some small path refactoring, and a better way to set the N3DS OC
patch.

Now I'm getting somewhere!

Wolfvak committed:
Overclocking for all titles

Only if the file "/injector/clock_all" exists, already have the
functionality in InjectMAN

Wolfvak committed:
Small modifications

Wolfvak committed:
Fixed N3DS clock, minor changes

Wolfvak committed:
Added support for 11.0 FIRM

Nothing special to be honest

Wolfvak committed:
Fixed for latest ctrulib

Pretty much a complete ripoff of
https://github.com/AuroraWright/Luma3DS/commit/b90b138766d6f076c76aa5505461793a6684086d
to be honest

dankzegriefer commented:
Conflict. Any idea what causes it?

ghost commented:
You have to revert back to https://github.com/dankzegriefer/3ds_injector/commit/6fa2e1a65a3ca974b1a3c5db2529d0bb113f7dbd, since that's were I forked. Otherwise, try force pushing.

dankzegriefer commented:
I'll do it when I wake up.

Wolfvak committed:
Update

Removed IFile in favor of FSFILE, fixed clock patch, rewrote a small
part of the README, first actual release.

Wolfvak committed:
Added RO sigpatches

Wolfvak committed:
Update for 11.1 FIRMs
